src/DssSpell.sol

```diff
@@ -106,226 +99,292 @@ contract DssSpellAction is DssAction {
-    address internal constant SPARK_AAVE_ORACLE              = 0x5Cd822d9a4421be687930498ec4B498EB972ad29;
-    address internal constant SPARK_ATOKEN_IMPL              = 0x35542cbc5730d5e39CF79dDBd8976ac984ca109b;
-    address internal constant SPARK_STABLE_DEBT_TOKEN_IMPL   = 0x571501be53711c372cE69De51865dD34B87698D5;
-    address internal constant SPARK_VARIABLE_DEBT_TOKEN_IMPL = 0xb9E6DBFa4De19CCed908BcbFe1d015190678AB5f;
-    address internal constant SPARK_INTEREST_RATE_STRATEGY   = 0xE7Fe5041ec55c229fb41fD9183E5bc24B5E34959;
-    address internal constant SPARK_TREASURY                 = 0x0D56700c90a690D8795D6C148aCD94b12932f4E3;
-    address internal constant SPARK_GNO_ORACLE               = 0xF7CBEF92ef380502dCe9DE8A654A5c1d2A513E78;
-    address internal constant GNO_MEDIANIZER                 = 0x0cd01b018C355a60B2Cc68A1e3d53853f05A7280;
-
-    address internal immutable REGISTRY                      = DssExecLib.reg();
-    address internal immutable MIP21_LIQUIDATION_ORACLE      = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-    address internal immutable ESM                           = DssExecLib.getChangelogAddress("MCD_ESM");
-    address internal immutable MCD_VAT                       = DssExecLib.vat();
-    address internal immutable MCD_JUG                       = DssExecLib.jug();
-    address internal immutable MCD_SPOT                      = DssExecLib.spotter();
-    address internal immutable MCD_DAI                       = DssExecLib.dai();
-
-    function onboardRWA014() internal {
-        bytes32 ilk      = "RWA014-A";
+    address internal constant RWA015_A_CUSTODY           = 0x65729807485F6f7695AF863d97D62140B7d69d83;
+
+    // Ilk registry params
+    uint256 internal constant RWA015_A_REG_CLASS_RWA = 3;
+
+    // RWA Oracle Params
+    uint256 internal constant RWA015_A_INITIAL_PRICE = 2_500_000;
+    string  internal constant RWA015_A_DOC             = "QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN";
+    uint48  internal constant RWA015_A_TAU           = 0;
+
+    // Remaining params
+    uint256 internal constant RWA015_A_LINE = 2_500_000;
+    uint256 internal constant RWA015_A_MAT  = 100_00;
+    // -- RWA015 END --
+
+    // Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
+    function _updateDoc(bytes32 ilk, string memory doc) internal {
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
+
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
+
+    function onboardRWA015A() internal {
```

Comment:
This should use the same convention as `_updateDoc`.
We either remove the leading `_` from both or add it here:
```suggestion
    function _onboardRWA015A() internal {
```